samples/excel/Basics/range.yaml

```diff
@@ -9,14 +9,40 @@ script:
     $('#setup').click(setup);
 
     $('#set-formulas').click(setFormulas);
```

Comment:
Pending the email I sent out about single vs. double quotes:  I wonder if we want to switch to all-double-quotes.  let's see what we decide on that email thread.